src/common/components/TabNameWithCount/TabNameWithCount.jsx

```diff
@@ -2,20 +2,24 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Icon } from 'antd';
 
+import EventTracker from '../EventTracker';
+
 class TabNameWithCount extends Component {
   render() {
     const { name, loading, count } = this.props;
     return (
-      <span>
-        <span>{name}</span>
-        <span className="ml1">
-          {loading ? (
-            <Icon className="ml1" type="loading" spin />
-          ) : (
-            count != null && <span>({count})</span>
-          )}
+      <EventTracker eventId={`${name}-Tab`}>
```

Comment:
what will happen if I click the active tab twice?

Reply:
it will send every time :/